src/sys/unix/selector/kqueue.rs

```diff
@@ -60,6 +49,7 @@ macro_rules! kevent {
             fflags: 0,
             data: 0,
             udata: $data as UData,
+            ..unsafe { mem::zeroed() }
```

Comment:
Could you add a note that this is needed for FreeBSD?

Reply:
Sure.  Except, looking at it again, I realize that I should've just deleted the fflags and data field initializers.